src/viadot/utils.py

```diff
@@ -18,7 +17,95 @@
 
 from viadot.exceptions import APIError, ValidationError
 from viadot.signals import SKIP
-from viadot.sources.base import Source
+
+try:
+    import pyspark.sql.dataframe as spark
+
+    from viadot.sources import Databricks
+except ImportError:
+    pass
+
+
+def _get_table_columns(
```

Comment:
I don't think this file should be changed at all; the solution to the Databricks import is [already implemented](https://github.com/dyvenia/viadot/blob/2.0/src/viadot/utils.py) in `2.0` branch (I added it in the mean time since I needed it for something else).

Reply:
[♻️ Refactored databricks related functions](https://github.com/dyvenia/viadot/pull/880/commits/c222d003180c954aa3cda6eed22c007fbd0c78a8)
Changed only spark import because its raise a exception when databricks is not installed